### DISCUSSION TITLE
[SOLVED] RSSGaurd will not fetch any feeds

### DISCUSSION THREAD

**Discussion by iamtherobin** [martinrotter/rssguard#625] [General]
I just installed latest RSSGuard on windows 7 x64 and imported my feeds from a OPML file. The feeds loaded and the metadata for the feeds can be fetched successfully. But when I fetch the articles for the feeds themself, it does not download anything. It just shows no feeds in the right side panel and the feed color changes (I assume indicating it has no articles).

<a href="https://imgur.com/GlhQ1SL"><img src="https://i.imgur.com/GlhQ1SLm.png" title="source: imgur.com" /></a>

**Comment by martinrotter** [comment]
> But when I fetch the articles

So you pressed "Fetch all" and nothing happened? Where and what file of RSS Guard did you download exactly? Did you change anything in settings?

**Comment by iamtherobin** [comment]
I tried https://github.com/martinrotter/rssguard/releases/download/4.1.2/rssguard-4.1.2-1f5157fd-win64.exe and https://github.com/martinrotter/rssguard/releases/download/4.1.2/rssguard-4.1.2-1f5157fd-nowebengine-win64.exe both gave me the same results.

I changed some setting like skins and global feed refresh times. I did not set anything in browser and email settings. In data storage, I tried all combinations of DB transactions and db in ram options, still nothing.

**Reply by martinrotter** [reply]
OK. Provide debug log.

https://github.com/martinrotter/rssguard/blob/master/resources/docs/Documentation.md#reprt

**Reply by iamtherobin** [reply]
https://zerobin.net/?c683606e7f244366#ww2cEn96AgvA5+6A7fyIu66/pYnDrISwRjJz3ewYMwI=

password is rsslog

**Comment by martinrotter** [comment]
You have some proxy? I see errors:

network: Setting specific downloader proxy, address: 'https://odysee.com/$/rss/@TimcastIRL:8', type: '3'.

core: Error 'QNetworkReply::HostNotFoundError' during fetching of new messages for feed 'https://www.youtube.com/feeds/videos.xml?channel_id=UC4BZtFgtCuHUt0p8J-XENiA'.

**Reply by iamtherobin** [reply]
I am not running a proxy. I tried no proxy and system proxy. Both no changes.

**Reply by iamtherobin** [reply]
Wait you're right.... I see now how the proxy setting for the account is different than the application setting... I accidentally pasted a feed URL when creating the account as a proxy.

**Reply by iamtherobin** [reply]
OK I removed the proxy setting from the account and can confirm it is working fine now. Thanks for the help. I would never have noticed that otherwise and would probably have just cleared the database and never realized each account sets its own proxy.

**Reply by martinrotter** [reply]
Yes, I will check the othe things you mentioned.

**Comment by iamtherobin** [comment]
I notice if I right click a feed and click fetch metadata, it gives an error popup with 'no host found' error. But if I click edit selected item, then click 'fetch it now' for the metadata, it will successfully fetch the metadata.

**Reply by martinrotter** [reply]
FIxed this anomaly, thanks.

**Comment by ghost** [comment]
I thought that I had a settings bug until I found this thread. I can't be convinced that having a global proxy setting and an "account" proxy setting is not a bad idea.
